### PR TITLE
Address dependabot and github action failures

### PR DESCRIPTION
**Pull request by ajbozarth**
This fixes some issues found by Dependabot and the default GitHub Actions that were enabled with the move to public GitHub

### CODE REVIEW

**Review by cbjuan — COMMENTED** [review]
Just one comment before approval

File: README-PyPi.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
 # Qiskit Code Assistant (Beta)
 
 > This experimental feature is only available, as of today, for some IBM Quantum users.
-  If you are not part of that cohort of users, you can still install this extension; however you will not be able to use the assistant.
-  The Qiskit Code Assistant is a beta release, subject to change.
+> If you are not part of that cohort of users, you can still install this extension; however you will not be able to use the assistant.
+> The Qiskit Code Assistant is a beta release, subject to change.
```
[the remark this answers is not in the document]
Comment (ajbozarth) [reply README-PyPi.md:5]: should we also remove (Beta) from the header of this file then?

[the remark this answers is not in the document]
Comment (cbjuan) [reply README-PyPi.md:5]: let's keep it beta for now

[the remark this answers is not in the document]
Comment (ajbozarth) [reply README-PyPi.md:5]: ok, so change this line but not the title?

[the remark this answers is not in the document]
Comment (cbjuan) [reply README-PyPi.md:5]: yes

**Review by cbjuan — APPROVED** [review]

**Pull request merged**